### DISCUSSION TITLE
In-game AI (ChatGBT)

### DISCUSSION THREAD

**Discussion by gettheboom** [chromiecraft/chromiecraft#5127] [Suggestions]
Could we have implemented an AI into Chromiecraft? Please see an example of ChatGBT being used on an RP server

https://www.youtube.com/watch?v=WH9cSj28GRc

**Comment by BeefSteaker** [comment]
Looks interesting. Probably something that could be handled client-side with an addon

**Reply by B4nanaboi** [reply]
I'm just here to tell, the responses in the video, not only fairly accurate, but super cool. It surely makes the world feel more alive and dynamic, when you can 'actually' talk to an NPC. Add text-to-speech in the applicable voice/accent and you've just created the coolest private server of all :) --- We've seen AI learn how to speak 99% perfectly like our president or any Hollywood star by just listening to their past recordings. If you could do that with NPC's, you could theoretically make them actually read out whatever chatGB(?)T writes out, or whatever quest text is applicable. --- Man, the possibilities are potentially endless, if only we had a person working fulltime on this -bonus- (but supercool) option :) Client-side addon could be an option indeed. If it could read quest text out loud, that would make it even more epic.
